lib/_computed/schedule.js

```diff
@@ -12,31 +11,34 @@ const phases = {
     ADOPT: "adopt",
     DEPRECATE: "deprecate",
     DECOMMISSION: "decommission"
-}
-module.exports = function(data) {
-    releaseAdoptionSchedule = data['releaseAdoptionSchedule']
+};
+
+const getReleaseAdoptionSchedule = data => {
+    const releaseAdoptionSchedule = data.releaseAdoptionSchedule ?? {};
     for (const tech in releaseAdoptionSchedule) {
-        let phase = phases.PENDING, adopted = false
-        techSchedule = releaseAdoptionSchedule[tech]
-        for (const item of techSchedule){
+        const techSchedule = releaseAdoptionSchedule[tech];
+        let phase = phases.PENDING, adopted = false;
+        for (let item of techSchedule) {
```

Comment:
why change `const` to `let`. Item isn't being reassigned
https://stackoverflow.com/a/58483224